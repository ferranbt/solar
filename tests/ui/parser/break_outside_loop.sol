function f() {
    break; //~^ERROR: `break` outside of a loop
}
