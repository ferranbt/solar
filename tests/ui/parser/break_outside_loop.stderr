error: `break` outside of a loop
  --> ROOT/tests/ui/parser/break_outside_loop.sol:LL:CC
   |
LL | / function f() {
LL | |     break;
LL | | }
   | |_^
   |

error: aborting due to 1 previous error

